codex/utils/timer.nim

```diff
@@ -35,6 +35,9 @@ proc timerLoop(timer: Timer) {.async: (raises: []).} =
       await sleepAsync(timer.interval)
   except CancelledError:
     discard # do not propagate as timerLoop is asyncSpawned
+  except CatchableError as err:
+    error "CatchableError in timer loop", name = timer.name, msg = err.msg
+  info "Timer loop has stopped", name = timer.name
```

Comment:
Is this really `info` worthy?

Reply:
Normally, the info only shows up when a timer is stopped. That's not a common thing, but not necessarily an error. However, if there are issues it's good to be able to see whether the timer was stopped.